source/kernel/trace_inode.h

```diff
@@ -0,0 +1,61 @@
+#ifndef TRACE_INODE_H_
+#define TRACE_INODE_H_
+
+#include "trace.h"
+
+/**
+ * @file
+ *
+ * @brief inodes tracer
+ *
+ * It provides tracing of following inodes properties:
+ * - Inode's name; It keeps an internal cache with information if given inode
+ * name has been already traced. If it hasn't, the inode name is traced.
```

Comment:
Please also add information that:
1. Each element of inode's full path is added as a separate FilenameEvent
2. Each inode's FilenameEvent is usually (see below) added only once (per cpu).
3. Due to internal cache limitations, some rarely accesed inodes' filenames may be traced more than once